packages/edit-site/src/components/global-styles/font-library-modal/upload-fonts.js

```diff
@@ -94,6 +115,22 @@ function UploadFonts() {
 		handleInstall( fontFacesLoaded );
 	};
 
+	/**
+	 * Checks if a file is a valid Font file.
+	 *
+	 * @param {file} file The file to be checked
```

Comment:
My reading of the MDN docs is that it's a File not a file. 
```suggestion
	 * @param {File} file The file to be checked.
	 * @return {boolean} Whether the file is a valid font file.
```